Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationXaml.bind

```diff
@@ -0,0 +1,357 @@
+                    Click="ShowNotificationWithButtonsDataTemplateButton_Click" />
+            
+            <Button x:Name="ShowNotificationWithDropShadowButton" 
+                    Margin="5"
+                    Content="Show notification with Drop Shadow (based on default template)"
+                    Click="ShowNotificationWithDropShadowButton_Click" />
+
+            <Button x:Name="ShowNotificationWithVSCodeTemplateButton" 
+                    Margin="5"
+                    Content="Show notification with Visual Studio Code template (info notification)"
+                    Click="ShowNotificationWithVSCodeTemplateButton_Click" />
+
+            <Button x:Name="DismissNotificationButton" 
+                    Margin="5"
+                    Content="Dismiss"
+                    Click="DismissNotificationButton_Click" />
+        </StackPanel>
+
+        <controls:InAppNotification x:Name="ExampleInAppNotification"
+                                    Content="This is a test message."
+                                    ShowDismissButton="@[ShowDismissButton:Bool:True]" />
+
+        <controls:InAppNotification x:Name="ExampleVSCodeInAppNotification"
+                                    MinHeight="35"
+                                    Background="#333333"
+                                    Foreground="White"
+                                    FontSize="14"
+                                    Padding="0"
+                                    BorderThickness="0"
+                                    ShowDismissButton="False"
+                                    VerticalAlignment="Top"
+                                    RenderTransformOrigin="0.5,0">
+            <controls:InAppNotification.ContentTemplate>
+                <DataTemplate>
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto" />
+                            <ColumnDefinition />
+                            <ColumnDefinition Width="Auto" />
+                        </Grid.ColumnDefinitions>
+
+                        <Grid Margin="10 5" Height="20" Background="#007ACC">
+                            <TextBlock Text="info"
+                                       Padding="5 0"
+                                       FontWeight="Bold"
+                                       VerticalAlignment="Center"
+                                       Foreground="White" />
+                        </Grid>
+
+                        <TextBlock Grid.Column="1" 
+                                   VerticalAlignment="Center"
+                                   Text="This a test message."
+                                   TextWrapping="NoWrap" TextTrimming="CharacterEllipsis" />
+
+                        <StackPanel Grid.Column="2" Orientation="Horizontal">
+                            <Button Content="Action 1" 
+                                    Click="Action1Button_Click"
```

Comment:
Trying to think how this click event will work in the new sample app...  

Since these are more geared to individual notifications vs a central service, I'd like to see an Opened/Opening event like FlyoutBase has.  Then we can grab the buttons in that event and register handlers for the sample app.